Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
+    var collectionView: UICollectionView!
+    var collectionViewHeight: CGFloat!
+    var dropdownView: DropdownView!
+    var facility: Facility!
+    var headerImageView: UIImageView!
+    var headerNameLabel: UILabel!
+    var headerOpenLabel: UILabel!
+    var headerView: UIView!
+    var headerViewHeight: CGFloat = 52
+    var height: CGFloat!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupDropdownView()
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(for facility: Facility) {
+        self.facility = facility
+        setupDropdownView()
+        setupConstraints()
```

Comment:
calling these two functions in the configure means that we would be reinitializing `headerView` and settings its constraints upon **every** single reuse. we should not be doing this. inside `configure` we should only be updating the properties and constraints that need to be updated